taprpc/taprootassets.proto

```diff
@@ -1492,6 +1505,32 @@ message OutPoint {
     uint32 output_index = 2;
 }
 
+// A transaction outpoint annotated with TAP-level asset metadata. It uniquely
+// identifies an asset anchored at a specific outpoint.
+message AssetOutPoint {
+    // The outpoint of the asset anchor, represented as a string in the
+    // format "<txid>:<vout>". The <txid> is the transaction ID of the UTXO,
+    // hex-encoded and byte-reversed (i.e., the internal little-endian
+    // 32-byte value is reversed to big-endian hex format) to match standard
+    // Bitcoin RPC and UI conventions.
+    string anchor_out_point = 1;
+
+    // The asset ID of the asset anchored at the outpoint.
+    AssetID asset_id = 2;
+
+    // The script key of the asset. This is the taproot output key that the
+    // asset is locked to.
+    oneof script_key {
+        // The script key in raw byte form. Use this field when interacting
+        // via gRPC.
+        bytes script_key_bytes = 4;
```

Comment:
Same question here: Why do we need both forms? On the CLI it should look the same. But in an itest I'd never easily know which field would actually be set by the RPC server (without looking up the code or calling the RPC).